src/accessibility/sonification/index.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+var c2m = require('chart2music');
+var Fx = require('../../components/fx');
+var Lib = require('../../lib');
+
+var codecs = require('./all_codecs').codecs;
+
+function initC2M(gd, defaultConfig) {
+    // TODO: should this be Lib.getGraphDiv()?
+    // TODO what is there besides a fullReset?
+    // TODO Do we need the capacity to add data (live listen?)
+    // TODO Do we need the capacity to reset all data
+    var c2mContext = gd._context._c2m = {};
+    c2mContext.options = Lib.extendDeepAll({}, defaultConfig.options);
+    c2mContext.info = Lib.extendDeepAll({}, defaultConfig.info);
+    c2mContext.ccOptions = Lib.extendDeepAll({}, defaultConfig.closedCaptions);
+
+    var labels = []; // TODO this probably needs to be stored in context-
+    // when data is updated this specific instance needs to be updated
+    // or the below function eneds to be reset to use the new instance of labels.
+    c2mContext.options.onFocusCallback = function(dataInfo) {
+        Fx.hover(gd, [{
+            curveNumber: labels.indexOf(dataInfo.slice),
+            pointNumber: dataInfo.index
+        }]);
+    };
+    // I generally don't like using closures like this.
+    // In this case it works out, we effectively treat 'labels'
+    // like a global, but it's a local.
+    // I'd rather the callback be given its c2m handler which
+    // could store extra data. Or bind c2mContext as `this`.
+
+    var titleText = gd._fullLayout.title?.text ?? 'Chart';
+    var ccElement = initClosedCaptionDiv(gd, c2mContext.ccOptions);
+
+    // TODO:
+    // I believe that title OR title.text could contain the information needed
```

Comment:
@archmoj Is that accurate that title or title.text could contain the information needed?

Reply:
The preferred way of setting a title is via `title.text` not `title`. 
Setting `title` to a string is discouraged(& deprecated) but may still widely used in our codes.
We may handle case of `title` in case title is set to a string instead of an object.

Reply:
We should drop support for that!